mllib/src/main/scala/org/apache/spark/mllib/tree/impl/TreePoint.scala

```diff
@@ -96,14 +96,12 @@ private[tree] object TreePoint {
    * Find bin for one (labeledPoint, feature).
    *
    * @param featureArity  0 for continuous features; number of categories for categorical features.
-   * @param isUnorderedFeature  (only applies if feature is categorical)
```

Comment:
Sounds good.  Can you please also remove it from labeledPointToTreePoint and not compute it in convertToTreeRDD?